src/components/Text.svelte

```diff
@@ -0,0 +1,12 @@
+<script>
+    export let body;
+    export let languageId;
+    export let breakAll;
+</script>
+
+{#if breakAll}
+    <div class="kui-body kui-text kui-text-break" tabindex="1" data-l10n-id="{languageId}" on:click>{body}</div>
```

Comment:
Here you can use `<slot />` instead of passing in body, and in the parent component you can do: `<Text>{body}</Text>`. See https://svelte.dev/tutorial/slots for more info :-)

Reply:
Just merge it and change it how you like it :)